src/Installer/redist-installer/targets/GenerateBundledVersions.targets

```diff
@@ -1247,7 +1247,9 @@ Copyright (c) .NET Foundation. All rights reserved.
     <PropertyGroup>
       <MinimumMSBuildVersionFile>$(RedistLayoutPath)sdk/$(Version)/minimumMSBuildVersion</MinimumMSBuildVersionFile>
       <BundledMSBuildPropsFileName>Microsoft.NETCoreSdk.BundledMSBuildInformation.props</BundledMSBuildPropsFileName>
-      <BundledMSBuildVersion>$(MSBuildVersion)</BundledMSBuildVersion>
+      <_ShippingMSBuildVersion>$(MicrosoftBuildVersion)</_ShippingMSBuildVersion>
+      <_ShippingMSBuildVersion Condition="$(_ShippingMSBuildVersion.Contains('-'))">$(MicrosoftBuildVersion.Split('-')[0])</_ShippingMSBuildVersion>
```

Comment:
It's possible for this version to be a preview version, but since we manipulate this value using System.Version, we need to turn it into something more .NET-friendly, so we drop everything after the `-`.